MEME/MEME/Artist/ArtistHome/ArtistHomeView/ArtistHomeViewController.swift

```diff
@@ -213,7 +219,59 @@ class ArtistHomeViewController: UIViewController {
         }
         return result.replacingOccurrences(of: "_", with: ":")
     }
+    
+}
+// MARK: - UITableViewDataSource
+extension ArtistHomeViewController : UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if let reservationData = reservationData {
+            fromTomorrowCount = 0
+            showDataCount = 0
+            for i in 0..<reservationData.count {
+                self.showDataCount += 1
+                if reservationStatusData[i] == 1{
+                    fromTomorrowCount += 1
+                    if fromTomorrowCount == 2 {
+                        break
+                    }
+                }
+            }
+            return self.showDataCount
+        }else {
+            return 0
+        }
+        
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // 셀 교체 필요
+        guard let resCell = artistReservationStatusTableView.dequeueReusableCell(withIdentifier: ArtistReservationStatusTableViewCell.identifier, for: indexPath) as? ArtistReservationStatusTableViewCell else { return UITableViewCell() }
+        resCell.makeUpNameLabel.text = reservationData[indexPath.row].makeupName
+        resCell.modelNameLabel.text = reservationData[indexPath.row].modelNickName
+        resCell.reservationDateLabel.text = formatDateString(op:1, reservationData[indexPath.row].reservationDate)
+        resCell.reservationTimeLabel.text = convertTimeString(reservationData[indexPath.row].reservationDayOfWeekAndTime.values.first!)
+        resCell.reservationPlaceLabel.text = reservationData[indexPath.row].shopLocation
+        resCell.reservationPriceLabel.text = "\(String(reservationData[indexPath.row].price))원"
+        // 버튼 태그로 index 전달
+        resCell.reservationManageBtn.tag = indexPath.row
```

Comment:
```suggestion
        let data = reservationData[indexPath.row]
        resCell.configure(data)
```

```
final class ReservationTableViewCell {
//,,UI 프로퍼티들,,
    func configure(_ cellModel: ReservationData) {
            makeUpNameLabel.text = cellModel.makeupName
            modelNameLabel.text = cellModel.modelNickName
            reservationDateLabel.text = formatDateString(op:1, cellModel.reservationDate)
          ....
    }
}
```

위와 같이 cell의 property에 직접 접근하지 말고, cell 내에 `configure()` 함수를 만들어서 cell 내부에서 property에 접근하는 방식은 어떨까요? 그리고 cell 내의 프로퍼티는 밖에서 접근할 수 없게 `private` 키워드를 붙이면 더 좋을 것 같습니다!
252 라인에 강제 언래핑은 옵셔널 바인딩으로 처리하는게 좋을 것 같습니다!!